.github/pull_request_template.md

```diff
@@ -4,21 +4,21 @@
 > - How does this pull request work? Need a brief introduction for the changed logic (optional)
 > - Separately describe each logical change and avoid lazy messages (optional)
 > - Describe any limitations of the current code (optional)
-> 
+>
 > **NOTE**: Do not assume reviewers understand the original issue
 
-## What are the type of the pull request? (mandatory)
+## What are the type of pull request? (mandatory)
 
 > The currently defined types are listed below, please pick one or more for this pull request
 > - New features (adds functionality)
 > - Improvement (an improvement to existing features)
 > - Bug fix (fixes an issue)
-> - Documentation (add some documentatation)
+> - Documentation (add some documentation)
 
 
 ## Does this pull request affect documentation (mandatory)
 
-> - If document has changed, please open an new issue and add the pull request link that solve the new issue here.
+> - If the document has changed, please open a new issue and add the pull request link that solves the new issue here.
```

Comment:
`If the document has been changed, please open a new issue and add the issue link here.`